collated/main/khooroko.md

```diff
@@ -0,0 +1,1424 @@
+# khooroko
+###### \java\seedu\address\commons\events\ui\ChangeThemeRequestEvent.java
+``` java
+/**
+ * Indicates a request to change theme
+ */
+public class ChangeThemeRequestEvent extends BaseEvent {
+
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName();
+    }
+}
+```
+###### \java\seedu\address\commons\events\ui\JumpToNearbyListRequestEvent.java
+``` java
+/**
+ * Indicates a request to jump to the list of nearby persons
+ */
+public class JumpToNearbyListRequestEvent extends BaseEvent {
+
+    public final int targetIndex;
+
+    public JumpToNearbyListRequestEvent(Index targetIndex) {
+        this.targetIndex = targetIndex.getZeroBased();
+    }
+
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName();
+    }
+
+}
+```
+###### \java\seedu\address\commons\events\ui\NearbyPersonNotInCurrentListEvent.java
+``` java
+/**
+ * Represents a selection change in the Nearby Person List Panel to a person that is not in the current Person List
+ * Panel.
+ */
+public class NearbyPersonNotInCurrentListEvent extends BaseEvent {
+
+
+    private final PersonCard newSelection;
+
+    public NearbyPersonNotInCurrentListEvent(PersonCard newSelection) {
+        this.newSelection = newSelection;
+    }
+
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName();
+    }
+
+    public PersonCard getNewSelection() {
+        return newSelection;
+    }
+}
+```
+###### \java\seedu\address\commons\events\ui\NearbyPersonPanelSelectionChangedEvent.java
+``` java
+/**
+ * Represents a selection change in the Nearby Person List Panel
+ */
+public class NearbyPersonPanelSelectionChangedEvent extends BaseEvent {
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.